### PR TITLE
Update the blog section

### PR DESCRIPTION
**Pull request by anthonydillon**
## Done
Update the blog section to match [the design](https://zpl.io/V1xDJ7k)

## QA
- Open the demo and go to /blog
- Check the page matches the design
- Check out the article page and see it looks good

Fixes https://github.com/canonical-web-and-design/snap-squad/issues/1990

## Screenshots
image

image

### CODE REVIEW

File: webapp/app.py [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import datetime
 import os
 
+import flask
```
Comment (tbille) [inline webapp/app.py:4]: you can remove :)

Comment (anthonydillon) [reply webapp/app.py]: It's gone

**Review by tbille — APPROVED** [review]
Looks good!

**Comment by solazio** [comment]
there seem to be an issue with loading some stuff on the big card
image

**Comment by anthonydillon** [comment]
Never seen that before. Anything in the console?

**Comment by tbille** [comment]
The images are not loading because the images are loaded from wordpress that is behind the VPN. We should proxy those images.

The second page should use this template: https://app.zeplin.io/project/606203889af57e12287b5308/screen/6088061241dda412e2fbf5c6

**Comment by anthonydillon** [comment]
Ah good point. I will need to look into how the image works for the smaller ones because the set the width and height and we need them larger for the features.

Could we just increase the posts per page? Otherwise I'll look into the second page on Monday.

File: templates/blog/partials/_blog-card.html [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  <div class="u-sv2 blog-p-card__header--{{ article.group.slug }}">
+    <h5 class="p-muted-heading">{{ article.group.name }}</h5>
+    <hr />
+  </div>
+  {% endif %}
+    <div class="p-card__content">
+    {% if article.image and article.image.source_url %}
+      <div class="u-crop--16-9">
+        <a href="/blog/{{ article.slug }}" aria-hidden="true" tabindex="-1">
+          {{ article.image.rendered|safe }}
```
[the remark this answers is not in the document]
Comment (tbille) [reply templates/blog/partials/_blog-card.html:12]: This will fix the VPN issue

**Comment by anthonydillon** [comment]
@tbille I have addressed the image issue and set the new layout for just page 1.

**Review by tbille — APPROVED** [review]

**Comment by spencerbygraves** [comment]
Thanks @anthonydillon a couple of things from me...

- There's a lot of space under the first large image, can we update in line with the design please
- There's no space between the smaller images and title, the ubuntu blog page is closer to the design
- The featured card title is using the soft link style, but all others are normal. I used soft for all but it should be consistent either way

Thanks

**Comment by anthonydillon** [comment]
@spencerbygraves I have addressed our comments.

**Pull request merged**